nservicebus/handlers/handler-ordering.md

```diff
@@ -21,10 +21,12 @@ The inference here is that the remaining handlers (not specified in the order) a
 
 <!-- import HandlerOrderingWithCode -->
 
-### Specifying First with ISpecifyMessageHandlerOrdering
+If you have been using `ISpecifyMessageHandlerOrdering` in a separate assembly scanned by NServiceBus we suggest to use `INeedInitialization`
```

Comment:
if we follow that approach then every configuration api needs some "BTW...INeedInitialization"  this is just not worth the effort. 

Also i dont know if the assumption that people using ISpecifyMessageHandlerOrdering  dont know about   INeedInitialization holds up.

as for

> Imagine people are using ISpecifyMessageHandlerOrdering in drop-in assemblies. Now that this interface is gone they are suddenly lost and might not remember to use INeedInitialization. The doco on this page hints them to 

this should be in the upgrade guide